falcon/testing/base.py

```diff
@@ -47,13 +47,13 @@ class TestBase(unittest.TestCase):
             can use to add a route to the API.
     """
 
-    def setUp(self):
+    def setUp(self, api=None, srmock=None):
```

Comment:
Normally this function is just called by the testing framework (unittest/testtools) and doesn't take any parameters. How will this be used?